### PR TITLE
chore: GH Presence

### PR DESCRIPTION
**Pull request by kratico**
Related #43 

Add the following files to configure GH presence
- `CONTRIBUTING.md`
- `CODE_OF_CONDUCT.md`
- `.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md`
- `.github/ISSUE_TEMPLATE/BUG_REPORT.md`
- `.github/PULL_REQUEST_TEMPLATE.md`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+
+name: 🐞 Bug Report
+about: Create a report to help us improve
+title: 🐞 <title>
```
Comment (tjjfvi) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.md:5]: Emoji in title adds visual noise and is redundant with label

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+
+name: 🚀 Feature Request
+about: Suggest an idea for this project
+title: 🚀 <title>
```
Comment (tjjfvi) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.md:5]: Emoji in title adds visual noise and is redundant with label

**Review by tjjfvi — COMMENTED** [review]
Looks generally good; a few notes:

File: .github/ISSUE_TEMPLATE/BUG_REPORT.md (hunk 1 above)
Comment (harrysolovay) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.md:5]: Agreed

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+# Contributing
+
+👍🎉 &nbsp; First off, thanks for taking the time to contribute! &nbsp; 🎉👍
```
Comment (harrysolovay) [inline CONTRIBUTING.md:3]: Same thing here

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+## Pull requests
+
+**For a pull request to be merged it must at least:**
+
+:white_check_mark: &nbsp; Pass CI
+
+:white_check_mark: &nbsp; Have one approving review
+
+:white_check_mark: &nbsp; Have the contributor(s) proud of hir contribution
```
[the remark this answers is not in the document]
Comment (harrysolovay) [reply CONTRIBUTING.md:27]: Nit: markdown checkboxes instead of emoji checkboxes?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+## Pull requests
+
+**For a pull request to be merged it must at least:**
+
+:white_check_mark: &nbsp; Pass CI
+
+:white_check_mark: &nbsp; Have one approving review
+
+:white_check_mark: &nbsp; Have the contributor(s) proud of their contribution
```
Comment (harrysolovay) [inline CONTRIBUTING.md:24]: Do we need this line?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+**Ideally, a good pull request should:**
+
+:clock3: &nbsp; Take less than 15 minutes to review
+
+:open_book: &nbsp; Have a meaningful description (describes the problem being solved) and commit messages that clarify intent
```
Comment (harrysolovay) [inline CONTRIBUTING.md:30]: Commit messages are somewhat irrelevant (we squash merge). Ideally they make some sense, but no need to be strict. PR titles, however, should follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).

File: CONTRIBUTING.md (hunk 5 above)
Comment (kratico) [reply CONTRIBUTING.md:24]: Probably not because it may be subjective

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+<!--- Describe your changes in detail -->
+
+**Related Issue**
+
+<!--- Please link to the issue here -->
+
+**Motivation and Context**
+
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
```
Comment (tjjfvi) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: All PRs should fix (or partially fix) a related issue, which should address the "motivation and context".

```suggestion
**Description**

<!--- Link to the open issue this addresses -->
Fixes #

<!--- Describe your changes in detail -->
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+**For a pull request to be merged it must at least:**
+
+:white_check_mark: &nbsp; Pass CI
+
+:white_check_mark: &nbsp; Have one approving review
+
+:white_check_mark: &nbsp; Have the PR title follow [conventional commit](https://www.conventionalcommits.org/)
+
+**Ideally, a good pull request should:**
```
Comment (tjjfvi) [inline CONTRIBUTING.md:24]: The items in this list should either consistently use or consistently not use periods

Comment (kratico) [reply CONTRIBUTING.md:24]: agreed

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+:open_book: &nbsp; Have a meaningful description (describes the problem being solved)
+
+:one: &nbsp; Introduce one feature or solve one bug at a time, for which an open issue already exists. In case of a project wide refactoring, a larger PR is to be expected, but the reviewer should be more carefully guided through it
+
+:jigsaw: &nbsp; Issues that seem too big for a PR that can be reviewed in 15 minutes or PRs that need to touch other issues should be discussed and probably split differently before starting any development
+
+:dart: &nbsp; Handle renaming, moving files, linting and formatting separately (not alongside features or bug fixes)
+
+:test_tube: Add tests for new functionality
```
Comment (tjjfvi) [inline CONTRIBUTING.md:36]:
```suggestion
:test_tube: &nbsp; Add tests for new functionality
```

**Review by tjjfvi — APPROVED** [review]

**Review by harrysolovay — APPROVED** [review]

**Pull request merged**